hack/values_eksctl.yaml

```diff
@@ -1,4 +1,3 @@
-enableVolumeSnapshot: true
 controller:
   logLevel: 5
 node:
```

Comment:
the serviceAccount.controller.create here also needs shuffling to controller.serviceAccount.create  : x